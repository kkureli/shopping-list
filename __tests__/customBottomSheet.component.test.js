import React from 'react';
import renderer from 'react-test-renderer';
import ActionButton from '../src/components/common/actionButton';

describe('Custom bottom sheet component', () => {
  it('renders Custom bottom sheet correctly', () => {
    const tree = renderer.create(<ActionButton />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
