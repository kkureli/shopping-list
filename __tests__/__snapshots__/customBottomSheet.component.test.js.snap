// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Custom bottom sheet component renders Custom bottom sheet correctly 1`] = `
<View
  accessible={true}
  collapsable={false}
  focusable={false}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "flexDirection": "row",
      "opacity": 1,
    }
  }
>
  <Text
    margin={null}
    padding={null}
    style={
      Object {
        "color": "rgb(0, 122, 255)",
        "fontFamily": "Montserrat",
        "fontSize": 16,
        "fontWeight": "bold",
      }
    }
    tertiary={false}
    theme={Object {}}
  />
</View>
`;
